src/templates/next/components/Siderail/Siderail.tsx

```diff
@@ -30,36 +30,46 @@ const SiderailMobile = ({
         )}
       </button>
 
-      {isExpanded &&
-        pages.map(({ url, title, isCurrent, childPages }) => {
-          return (
-            <div
-              className={`border-b border-divider-medium ${Paragraph[3]} text-content`}
-            >
-              <LinkComponent
-                href={url}
-                className={`${
-                  isCurrent ? "font-bold" : ""
-                } block py-4 px-5 hover:bg-interaction-sub active:underline active:underline-offset-2 `}
+      {isExpanded && (
+        <ul>
+          {pages.map(({ url, title, isCurrent, childPages }) => {
+            return (
+              <li
+                className={`border-b border-divider-medium ${Paragraph[3]} text-content`}
               >
-                {title}
-              </LinkComponent>
-              {isCurrent &&
-                childPages?.map(({ title, url }, index) => {
-                  return (
-                    <LinkComponent
-                      href={url}
-                      className={`block py-1.5 pl-12 hover:bg-interaction-sub active:underline active:underline-offset-2 ${
-                        index === childPages.length - 1 && "pb-4"
-                      }`}
-                    >
-                      {title}
-                    </LinkComponent>
-                  )
-                })}
-            </div>
-          )
-        })}
+                {isCurrent ? (
+                  <p className="font-bold block py-4 px-5">{title}</p>
```

Comment:
nit: can add an `aria-current="page"` in the `<li>` to indicate that the item represents the current page.

I think got a few other accessibility stuff to fix but going to ignore for now since this is mostly design fixes + we should use a proper accessibility checker later for all our components.